lib/rubocop/cop/rails/create_table_with_timestamps.rb

```diff
@@ -80,7 +84,9 @@ def on_send(node)
         private
 
         def time_columns_included?(node)
-          timestamps_included?(node) || created_at_or_updated_at_included?(node)
+          timestamps_included?(node) ||
+            created_at_or_updated_at_symbol_included?(node) ||
+            created_at_or_updated_at_string_included?(node)
```

Comment:
By writing `def_node_search` as below, this code can reduce the conditional expression.

```diff
diff --git a/lib/rubocop/cop/rails/create_table_with_timestamps.rb b/lib/rubocop/cop/rails/create_table_with_timestamps.rb
index b59b9d4eb..bf3de5a8a 100644
--- a/lib/rubocop/cop/rails/create_table_with_timestamps.rb
+++ b/lib/rubocop/cop/rails/create_table_with_timestamps.rb
@@ -58,12 +58,10 @@ module RuboCop
           (send _var :timestamps ...)
         PATTERN

-        def_node_search :created_at_or_updated_at_symbol_included?, <<-PATTERN
-          (send _var :datetime (sym {:created_at :updated_at}) ...)
-        PATTERN
-
-        def_node_search :created_at_or_updated_at_string_included?, <<-PATTERN
-          (send _var :datetime (str {"created_at" "updated_at"}) ...)
+        def_node_search :created_at_or_updated_at_included?, <<-PATTERN
+          (send _var :datetime
+            {(sym {:created_at :updated_at})(str {"created_at" "updated_at"})}
+              ...)
         PATTERN

         def on_send(node)
@@ -84,9 +82,7 @@ module RuboCop
         private

         def time_columns_included?(node)
-          timestamps_included?(node) ||
-            created_at_or_updated_at_symbol_included?(node) ||
-            created_at_or_updated_at_string_included?(node)
+          timestamps_included?(node) || created_at_or_updated_at_included?(node)
         end
       end
     end
```

Reply:
aha! so that's how you do it, thank you!